docs/modules/ref_prebuilt-keycloak-server.adoc

```diff
@@ -0,0 +1,20 @@
+
+[id='prebuilt-keycloak-server_{context}']
+= {Thorntail} Keycloak Server
+
+https://www.keycloak.org/ [Keycloak] is an open source identity and access management solution.
```

Comment:
Space between the link and `[Keycloak]` is not OK I guess?

Reply:
Good catch, it is even visible in the attached image, missed it